orthwconfig-template

```diff
@@ -95,7 +95,7 @@ scandb_password=""
 ort_docker_registry_server=""
 ort_docker_registry_username=""
 ort_docker_registry_password=""
-ort_docker_image="registry.gitlab.com/oss-review-toolkit/ort-gitlab-ci/ort:latest"
+ort_docker_image="ghcr.io/oss-review-toolkit/ort-extended:latest"
```

Comment:
Should we wait until this is renamed to `ort-full` as determined by the [poll](https://ort-talk.slack.com/archives/C9NNJ54B1/p1695977688303359) in order to avoid another renaming?

Reply:
I'd be ok with updating it again, so that we have a more reasonable default config already now.